src/lightning/pytorch/loggers/neptune.py

```diff
@@ -551,7 +578,7 @@ def after_save_checkpoint(self, checkpoint_callback: Checkpoint) -> None:
     def _get_full_model_name(model_path: str, checkpoint_callback: Checkpoint) -> str:
         """Returns model name which is string `model_path` appended to `checkpoint_callback.dirpath`."""
         if hasattr(checkpoint_callback, "dirpath"):
-            expected_model_path = f"{checkpoint_callback.dirpath}{os.path.sep}"
+            expected_model_path = _sanitize_expected_model_path(checkpoint_callback.dirpath)
```

Comment:
Could this simply use https://docs.python.org/3/library/os.path.html#os.path.normpath?